test/test-window.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env mocha */
```

Comment:
Nit: all of the other tests have the `-test` suffix

Reply:
Fixed.